cumulus/steps/ec2/port.py

```diff
@@ -0,0 +1,31 @@
+from troposphere import elasticloadbalancingv2 as alb
+from troposphere import (
+    Ref, Join, ec2)
+from cumulus.chain import step
+
+
+class Port(step.Step):
```

Comment:
The name here is too generic. This is specifically a class for ASG--> ALB  traffic. 

I think it's ok to nuke this class and have this done directly where the target group is added.   Likely in the consumer code there are ports being passed into a few places.